notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>BigQuery DataFrames, not BigFrames DataFrames</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done.</p><p> </p><p> </p>